packages/office-ui-fabric-react/src/components/Rating/docs/new/RatingBestPractices.md

```diff
@@ -0,0 +1,9 @@
+### Layout
+
+- Make it clear which product the rating pertains to by making sure the layout and grouping are clear when several products are on the page.
```

Comment:
Should this say "product" or something more general like "item"? Ratings aren't used exclusively for products.